gravitee-apim-portal-webui-next/src/components/pagination/pagination.component.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2025 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, computed, EventEmitter, input, Output, Signal } from '@angular/core';
+import { MatButton, MatIconButton } from '@angular/material/button';
+import { MatIcon } from '@angular/material/icon';
+
+interface PaginationVM {
+  hasPreviousPage: boolean;
+  hasNextPage: boolean;
+  currentPage: number;
+  totalPages: number;
+}
+
+@Component({
+  selector: 'app-pagination',
+  standalone: true,
+  imports: [MatButton, MatIcon, MatIconButton],
+  templateUrl: './pagination.component.html',
+  styleUrl: './pagination.component.scss',
+})
+export class PaginationComponent {
+  @Output()
+  selectPage = new EventEmitter<number>();
```

Comment:
what about using the new angular outputs ?

```suggestion
  selectPage = output<number>();
```